src/common/Loading/styled.js

```diff
@@ -0,0 +1,25 @@
+import styled, { keyframes } from "styled-components";
+import { ReactComponent as Spiner } from "../../images/icon-spinner.svg"
+
+const rotate = keyframes`
+  from {
+    transform: rotate(0deg);
+}
+  to {
+    transform: rotate(360deg);
+}
+`;
+
+export const Wrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+`;
+
+export const StyledSpiner = styled(Spiner)`
+    animation: ${rotate} 1s liner infinite;
+    width: 91px;
+    height: 91px;
+
+`;
```

Comment:
Nie wiem czy tylko u mnie, ale ten spinner nie chce się kręcić, natomiast z pomocą internetu napisałem coś takiego:
```
export const StyledSpiner = styled(Spiner)`
    width: 91px;
    height: 91px;
    animation: move-spinner 4s linear infinite;

    @keyframes move-spinner {
      from {
        transform: rotate(0deg);
      }
      to {
        transform: rotate(360deg);
      }
    }
`;
```
i wygląda na to że działa. Możesz tego użyć, albo coś w tamtym pozmieniać 😜